projects/packages/videopress/src/class-initializer.php

```diff
@@ -193,14 +195,29 @@ public static function register_videopress_video_block() {
 		// Pick the block name straight from the block metadata .json file.
 		$videopress_video_block_name = $videopress_video_metadata->name;
 
-		// Register and enqueue scripts used by the VideoPress block.
-		Block_Editor_Extensions::init( $videopress_video_metadata );
+		/*
+		 * Use the videopress/video editor script handle to localize enqueue scripts.
+		 * @see https://developer.wordpress.org/reference/functions/generate_block_asset_handle
+		 */
+		$script_handle = generate_block_asset_handle( $videopress_video_block_name, 'editorScript' );
+
+		// Register and enqueue scripts used by the VideoPress video block.
+		Block_Editor_Extensions::init( $script_handle );
 
 		// Do not register if the block is already registered.
 		if ( \WP_Block_Type_Registry::get_instance()->is_registered( $videopress_video_block_name ) ) {
 			return;
 		}
 
-		register_block_type( $videopress_video_metadata_file );
+		Assets::register_script(
+			$script_handle,
+			'../build/block-editor/blocks/video/index.js',
+			__FILE__,
+			array(
+				'in_footer'  => true,
+				'textdomain' => 'jetpack-videopress-pkg',
+			)
+		);
+		Assets::enqueue_script( $script_handle );
```

Comment:
While I'm not very familiar with the block registration, I think this should still be a call to `register_block_type()` rather than `Assets::enqueue_script()` so everything gets registered properly. As it is, the script is getting enqueued but as far as I can tell `WP_Block_Type_Registry` never gets told about it.

Of course, for that to work the json file would need updating to point to the handle registered just above instead of having "file:index.js". So probably better to use a static handle rather than `generate_block_asset_handle()`.